### ISSUE TITLE
`comma-spacing` can't handle a trick with generics

### ISSUE THREAD

**Issue by havenchyk** [typescript-eslint/typescript-eslint#1457]
I didn't find the rule for eslint `comma-spacing`, it's a main reason to create this issue.

**Repro**

```TS
const MyComponent = <T,>(props: Props<T>) => { return <AnotherComponent {...props} /> }
```

**Expected Result**
I presume there should be a rule that replaces eslint `comma-spacing` rule and allow `,` for the next code: `<T,>`

**Actual Result**
There is no such rule :) 

**Additional Info**
This trick is needed for working together with TypeScript and React in `.tsx` file, otherwise compiler thinks it's a JSX/TSX code, not generic.

https://github.com/microsoft/TypeScript/issues/15713#issuecomment-499474386

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `2.15.0` |
| `@typescript-eslint/parser`        | `2.15.0` |
| `TypeScript`                       | `3.7.4` |
| `ESLint`                           | `6.8.0` |
| `node`                             | `13.3.0` |
| `npm`                              | `6.13.6` |

**Comment by havenchyk** [comment]
Wow, that was fast, thank you guys @a-tarasyuk @bradzacher
